mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -2501,12 +2507,37 @@ int main(int argc, char **argv) {
     }
     module = moduleRef.release();
 
+    if (!perfConfig.empty()) {
+      WalkResult findGemmOp = module->walk(
+          [&](rock::RockGemmWrapperInterface gemmOp) -> WalkResult {
+            OpBuilder b(gemmOp.getContext());
+            gemmOp->setAttr("perf_config", b.getStringAttr(perfConfig));
+            return WalkResult::interrupt();
+          });
+      if (!findGemmOp.wasInterrupted()) {
+        llvm::errs() << "Cannot find a Gemm kernel for perf_config\n";
```

Comment:
fyi, RockGemmWrapperInterface includes convolutions. When we find more ops to tune, we need a new interface such as `RockTunableOpInterface`.